.mvn/maven.config

```diff
@@ -1,2 +1,10 @@
 --strict-checksums
-
+-Daether.syncContext.named.factory=file-lock
+-Daether.syncContext.named.nameMapper=file-hgav
+-Daether.syncContext.named.retry=5
+-Daether.syncContext.named.time=30
```

Comment:
What's the unit of this?

Reply:
aether.syncContext.named.time.unit by default is `TimeUnit.SECONDS`

Reply:
`aether.syncContext.named.time` already has a default of `30`, remove it or add the unit too to make it easier to read.